Jenkinsfile

```diff
@@ -3,9 +3,9 @@
 @Library('OGJenkinsLib@3.7.0') _
 
 final GIT_REPOSITORY_NAME = 'amazon-eks-ami'
-final KUBERNETES_VERSION = '1.13.7'
+final KUBERNETES_VERSION = '1.15.10'
```

Comment:
also Bump OGJenkinslib to 3.18  on line 3

Reply:
bumped 3.18.0